pkg/jsonedit/transform.go

```diff
@@ -64,16 +65,21 @@ func transformFromComplexPath(jsonBytes []byte, complexPath string, transformChi
 			return unpacked, fmt.Errorf("Node %s not a list", parent)
 		}
 		items := make([]interface{}, 0)
-		for _, v := range parentList {
-			jsonBytes, err := json.Marshal(&v)
+		for i := range parentList {
+			var err error
+			jsonBytes, err = json.Marshal(&parentList[i])
 			if err != nil {
 				return unpacked, err
 			}
 			jsonBytes, err = Transform(jsonBytes, []string{child}, transformChildFn)
+			if err != nil {
```

Comment:
Most likely we want to continue here: if one item has an error, we do not want to skip the whole thing.

Reply:
but don't we want to log as warning/error?

Reply:
most likely, yes we would, but if logger is not already there it might be out of scope in this PR. As currently this is already not logging, we can add it later in a following PR as I feel bad for @danmanor having to fix this stuff :sweat_smile: